backend/Posts/serializers.py

```diff
@@ -4,7 +4,7 @@
 # To serialise the post which validates data from the front end
 class PostSerializer(serializers.ModelSerializer):
     class Meta:
-        model = Post
+        #model = Post
```

Comment:
this looks like an issue, is it possible to just leave changing this file out of the pr?